k8s.gcr.io/images/k8s-staging-build-image/images.yaml

```diff
@@ -392,22 +394,32 @@
     "sha256:ff6e2f3683e7d284674ed18341fc898060204e8c43c9b477e13c6f7faf3e66d4": ["buster-v2.0.0"]
 - name: go-runner-amd64
   dmap:
+    "sha256:1ca263fc76083cc779310b665e6f8c5b37078dc7935d9b1ef084ca27ac3c6847": ["v2.3.1-go1.17.7-bullseye.0"]
+    "sha256:2e6c3bd28ae0aa3cb25171888c671b00fa48999f744d30abacba9bd1f22912d9": ["v2.3.1-go1.16.14-buster.0"]
     "sha256:2fd65994929a83ece98cdfb3ff1ab56ba5b13ac8c25ed6028df98e0cb2137751": ["v2.3.1-go1.16.13-buster.0"]
     "sha256:6078e5b3ef3fd90d8b7813a01064d736fd8367bdabd72eabfb76ddf7972ba844": ["v2.3.1-go1.17.6-bullseye.0"]
 - name: go-runner-arm
   dmap:
     "sha256:40479777cbfedd23a93f5638d0b2af7e2f58cdd1ccdbcd0655878ae46612a82b": ["v2.3.1-go1.17.6-bullseye.0"]
+    "sha256:c9a5183e4382112f3b04fb4de3b54bbc8e855f78ba100d5751c01c57622bd5c0": ["v2.3.1-go1.16.14-buster.0"]
     "sha256:f5209b750a6d791af4d77050f7ba5b57c61711dc888d6a981300d231c492c72a": ["v2.3.1-go1.16.13-buster.0"]
+    "sha256:fdb24b9d4701383101b52f096cb1b928c4e1fac244c687bc6e4a986e5b4770db": ["v2.3.1-go1.17.7-bullseye.0"]
```

Comment:
same: we can remove those, the existing ones pass thru, but we don't need to promote those